app/views/order_items/index.html.erb

```diff
@@ -1,32 +1,43 @@
-<h1>Your Cart</h1>
+<div class="panel panel-warning">
+  <div class="panel-body">
+    <h2>Your Cart  <span class="totalcart">Total:<%= @order.order_total%></span>
+</h2>
+    <!-- <% if @order.nil? %>
+      <p>No items in cart. </p>
+    <% else %> -->
+    <ul>
+      <% @order_items.each do |item| %>
+        <% product = Product.find(item.product_id) %>
+        <li>
+          <%= image_tag product.image, height: '100', width: '100', class: "productpic" %>
```

Comment:
Sorry ignore them! Didn't realize you were still receiving them. There was an instance variable @place and I inadvertently tagged you in a comment. Won't happen again. 

Sent from my iPhone

> On Jun 13, 2016, at 4:48 PM, Place <notifications@github.com> wrote:
> 
> In app/views/order_items/index.html.erb:
> 
> > @@ -1,32 +1,43 @@
> > -<h1>Your Cart</h1>
> > +<div class="panel panel-warning">
> > +  <div class="panel-body">
> > +    <h2>Your Cart  <span class="totalcart">Total:<%= @order.order_total%></span>
> > +</h2>
> > +    <!-- <% if @order.nil? %>
> > +      <p>No items in cart. </p>
> > +    <% else %> -->
> > +    <ul>
> > +      <% @order_items.each do |item| %>
> > +        <% product = Product.find(item.product_id) %>
> > +        <li>
> > +          <%= image_tag product.image, height: '100', width: '100', class: "productpic" %>
> Who are you, why am I receiving emails from you, and did I sign up to receive them? If so, when? Respond today. Thank you. Thomas Murphy
> …
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>